deployment/network-operator/Chart.yaml

```diff
@@ -15,7 +15,7 @@ dependencies:
 - condition: nfd.enabled
   name: node-feature-discovery
   repository: "http://kubernetes-sigs.github.io/node-feature-discovery/charts"
-  version: 0.10.1
+  version: 0.13.2
```

Comment:
bumping the version will now trigger creation of NFD CRDs when deploying NFD with network operator.
we support of upgrading CRDs via helm hooks.

see: https://github.com/Mellanox/network-operator/blob/0ed7f5c9fc380091b893ad43be112ab0dceaac3d/deployment/network-operator/templates/upgrade-crd.yaml#L71

crds are copied to network-operator container image and the hook runs with this image and kubectl applies them
see: https://github.com/Mellanox/network-operator/blob/0ed7f5c9fc380091b893ad43be112ab0dceaac3d/Dockerfile#L41

we should do the same for NFD CRDs.

Reply:
note: to get the CRDs of NFD you can run `helm dependency update` to download the chart.

alternatively we can store the dependent chart in project.